docs/views.md

```diff
@@ -53,3 +53,14 @@ system and can no longer be used to authenticate.
 **Note** It is not recommended to alter the Logout views. They are designed
 specifically for token management, and to respond to Knox authentication.
 Modified forms of the class may cause unpredictable results.
+
+## TokenRefreshView
+Optional view for refreshing the expiration date of a single token.
+It accepts an empty `POST` request against `/refresh` to increase the `expiry` field of the
+token by the amount set in `TOKEN_TTL`.
+This endpoint is only exposed if `AUTO_REFRESH` and `TOKEN_TTL` are enabled. 
+
+The functionality can be disabled entirely by setting `DISABLE_REFRESH_ENDPOINT=True` in `REST_KNOX` settings.
```

Comment:
We could get rid of the logic explained above by just having a `ENABLE_REFRESH_ENDPOINT` setting.
It would be easier to explain and would also make the code simpler.

The explanation would boil down to:

> If you want the refresh endpoint, just opt-in by setting `ENABLE_REFRESH_ENDPOINT` to `True`.

Simple, straight, obvious, no magic.